src/Quartz.Tests.Unit/Quartz.Tests.Unit.csproj

```diff
@@ -5,6 +5,9 @@
     <Nullable>disable</Nullable>
     <NoWarn>SYSLIB0011;CS0618</NoWarn>
     <EnableUnsafeBinaryFormatterSerialization>true</EnableUnsafeBinaryFormatterSerialization>
+
+    <!-- Microsoft.Extensions.TimeProvider.Testing 8.6.0 doesn't support net472 and has not been tested with it... -->
+    <SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings>
```

Comment:
Microsoft.Extensions.TimeProvider.Testing has net462 target and seems to work when running with net 472. Not sure if we want to disable all the warnings as SuppressTfmSupportBuildWarnings is not M.E.T.T specific.